WordPress/Classes/ViewRelated/Blog/SiteSettingsViewController.m

```diff
@@ -245,12 +253,48 @@ - (SettingTableViewCell *)geotaggingCell
     return _geotaggingCell;
 }
 
+- (SettingTableViewCell *)defaultCategoryCell
+{
+    if (_defaultCategoryCell){
+        return _defaultCategoryCell;
+    }
+    _defaultCategoryCell = [[SettingTableViewCell alloc] initWithLabel:NSLocalizedString(@"Default Category", @"Label for selecting the default category of a post")
+                                                           editable:YES
+                                                    reuseIdentifier:nil];
+    return _defaultCategoryCell;
+}
+
+- (SettingTableViewCell *)defaultPostFormatCell
+{
+    if (_defaultPostFormatCell){
+        return _defaultPostFormatCell;
+    }
+    _defaultPostFormatCell = [[SettingTableViewCell alloc] initWithLabel:NSLocalizedString(@"Default Post Format", @"Label for selecting the default post format")
+                                                              editable:YES
+                                                       reuseIdentifier:nil];
+    return _defaultPostFormatCell;
+}
+
+
 - (UITableViewCell *)tableView:(UITableView *)tableView cellForWritingSettingsAtRow:(NSInteger)row
 {
-    if (row == SiteSettingsWritingGeotagging) {
-        UISwitch *geotaggingSwitch =  (UISwitch *)self.geotaggingCell.accessoryView;
-        geotaggingSwitch.on = self.geolocationEnabled;
-        return self.geotaggingCell;
+    switch (row) {
+        case (SiteSettingsWritingGeotagging):{
+            UISwitch *geotaggingSwitch =  (UISwitch *)self.geotaggingCell.accessoryView;
+            geotaggingSwitch.on = self.geolocationEnabled;
+            return self.geotaggingCell;
+        }break;
```

Comment:
I don't think you need that `break` since you're calling `return`. In any case, if it's there it should be on it's own line. There are a couple more for the other `case` blocks